plugins/ipam/ipstore/ipstore.go

```diff
@@ -52,6 +51,7 @@ type IPAllocator interface {
 	Assign(string, string) error
 	Update(string, string) error
 	Release(string) error
+	ReleaseByValue(string) (string, error)
```

Comment:
Can you rename this as `ReleaseByID` since that's the abstraction provided by this interface?